### DISCUSSION TITLE
my mobile

### DISCUSSION THREAD

**Discussion by Trizkitz** [community/community#20196] [Mobile]
my question is !!

how would I start codeing? is GitHub - [ ] the base of everything software?~~~~****

**Comment by mmvanheusden** [comment]
GitHub is only a platform for "backing up your code", and coding others code. You should start learning an easy language like HTML, you don't need GitHub yet

**Comment by diegoleondev** [comment]
Hi @Trizkitz, GitHub offers an integrated development environment, you can visit https://github.dev/github/dev to get started.

GitHub is a version control tool, it helps to maintain order in projects, especially those that involve many people.
